src/test/java/org/folio/tlib/postgres/PgCqlQueryTest.java

```diff
@@ -0,0 +1,125 @@
+package org.folio.tlib.postgres;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class PgCqlQueryTest {
+
+  @Test
+  public void testSimple() {
+    PgCqlQuery pgCqlQuery = PgCqlQuery.query();
+    Assert.assertNull(pgCqlQuery.getWhereClause());
+    pgCqlQuery.parse(null);
+    Assert.assertNull(pgCqlQuery.getWhereClause());
+
+    pgCqlQuery.addField(new PgCqlField("dc.title", "title", PgCqlField.Type.TEXT));
+
+    pgCqlQuery.parse("dc.Title==value");
+    Assert.assertEquals("title = 'value'", pgCqlQuery.getWhereClause());
+  }
+
+  static String ftResponse(String column, String term) {
+    return "to_tsvector('english', " + column + ") @@ plainto_tsquery('english', '" + term + "')";
+  }
+
+  @Test
+  public void testQueries() {
+    String[][] list = new String[][] {
```

Comment:
There is tooling support for parameterized tests resulting in better reporting both in IDE and in maven output:
* JUnit 4: https://github.com/Pragmatists/JUnitParams (additional dependency)
* JUnit 5: https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests (built in)

Why does mod-eusage-reports still use JUnit 4?

Reply:
Because I prefer Junit 4 over Junit 5 when using Vert.x ? Is something wrong with Junit 4?